### ISSUE TITLE
How can I set RE-NET w. GT to reproduce experiment

### ISSUE THREAD

**Issue by JuliaGast** [INK-USC/RE-Net#62]
Hello,
In your paper, table 2 you are showing an interesting ablation study, where you show also RE-NET w. GT.
I would like to understand how, in the code, I can set this feeding of the GT.

As far as I understand, I would have to feed after each predicted timestep, the ground truth graph, instead of the predicted graph, before going to the next timestep, right?

**Is there a configuration parameter to set? If yes: which one?**

Or do I directly need to modify the code?
If yes, I assume somewhere in model.py predict() I would have to feed the gt_graph instead of the predicted graph, is this correct? and what exactly would I have to modify?

Looking forward to your reply
Kind Regards
Julia

**Comment by woojeongjin** [comment]
Yes you should feed the ground-truth graphs instead of predicted graphs.

**Comment by GastJulia** [comment]
thanks for your reply. 

I assume I would have to feed the ground truth in model.py predict() 
for self.graph_dict[self.latest_time.item()] instead of 
![image](https://user-images.githubusercontent.com/58255222/185311411-39a07f7d-4231-487c-b7ab-6d8bade48afa.png)

Is this correct? Are there more modifications necessary?

**Comment by JuliaGast** [comment]
Thank you for your reply.

I assume I would have to feed the ground truth in model.py predict() at self.graph_dict[self.latest_time.item()] instead of:
 
![image](https://user-images.githubusercontent.com/41183430/185319583-a6fce1b4-1f47-42a8-a44f-bf904407c4c9.png)

Is this correct? 
Are there any other modifications to be done?

**Comment by binchen4110** [comment]
> Thank you for your reply.
> 
> I assume I would have to feed the ground truth in model.py predict() at self.graph_dict[self.latest_time.item()] instead of:
> 
> ![image](https://user-images.githubusercontent.com/41183430/185319583-a6fce1b4-1f47-42a8-a44f-bf904407c4c9.png)
> 
> Is this correct? Are there any other modifications to be done?
The same question. Do you succeed?
